src/Rules/HtmlXmlCompatibilityHelper.cs

```diff
@@ -30,11 +30,11 @@ internal static class HtmlXmlCompatibilityHelper
         // the empty elements and replace them with elements with closing tags
         // e.g. <br>  =>  <br/>
         // e.g. <col span="123">  =>  <col span="123"/>
-        private static readonly Regex CleanCol = new Regex("(?<element>(<col\\s*)|(col\\s+[^/^>]*))>",
+        private static readonly Regex CleanCol = new Regex("(?<element>(<\\s*col\\s*)|(<\\s*col\\s+[^/^>]*))>",
```

Comment:
Is the `\\s*` needed here?

Reply:
it would work in case of something like < col>. I would say it would work for now without it but I can't think of any scenario that this would be harmful.